web/docs/database/operations/queries.md

```diff
@@ -0,0 +1,666 @@
+---
+title: Queries
+---
+
+import { Required } from '@site/src/components/Required';
+
+We'll explain what Queries are and how to use them. If you're looking for a detailed API specification, skip ahead to the [Options Reference](#options-reference).
+
+You can use Queries to fetch data from the server. They shouldn't modify the server's state.
+Fetching all comments on a blog post, a list of users that liked a video, information about a single product based on its ID... All of these are perfect use cases for a Query.
+
+:::tip
```

Comment:
I would maybe remove this note 🤷  let them read, they skim over the similar bits?

Reply:
Talked about it, decided to leave it for now.